index.bs

```diff
@@ -1082,6 +1082,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
         This is done in order to match the behavior of pervasively deployed ambient credentials (e.g., cookies, [[RFC6265]]).
         Please note that this is a greater relaxation of "same-origin" restrictions than what
         [=document.domain=]'s setter provides.
+
+        These restrictions on origin values apply to [=WebAuthn Clients=]. Other types of FIDO clients may use different origin URI formats, and have different rules for binding the origin URI to a [=Relying Party Identifier=], however the definition of these is outside the scope of this specification.
```

Comment:
>  These restrictions on origin values

(nit: RP IDs aren't origins. Also, RP IDs as defined here aren't URIs, I think&mdash;although URI is pretty general so not totally sure about that.)